platform.md

```diff
@@ -1003,6 +1018,9 @@ Where:
 [run-image]
  image = "<image>"
  mirrors = ["<mirror>", "<mirror>"]
+[build-image]
+ stack-id = "<string>"
+ mixins = ["jq", "libgc", "libpq"]
```

Comment:
```suggestion
 mixins = ["<mixin name>"]
```
`jq` etc. are very example-y and we generally don't do that in the data formats. See https://github.com/buildpacks/spec/blob/main/buildpack.md#buildpacktoml-toml

Reply:
Removed in 6480d2e975ba81d5b61d681f0177539408c834b8